CMakeLists.txt

```diff
@@ -98,16 +96,22 @@ set(CMAKE_CXX_STANDARD 14)
 # Set up flags and dependencies:
 #----------------------------------------------------------
 
+# Note: any target dependency of arbor needs to be explicitly added
+# to the 'export set', even the private ones, and this must be done
+# in the same CMakeLists.txt in which the target is defined.
+
 # Interface library `arbor-private-deps` collects dependencies, options etc.
 # for the arbor library.
 
 add_library(arbor-private-deps INTERFACE)
+install(TARGETS arbor-private-deps EXPORT arbor-targets)
```

Comment:
This will install the dependencies that you have taken great care to corral (because CMake). What exactly will be installed?

Reply:
Only the target definition in the CMake-generated config files; there are no files associated with these targets, and so nothing else will be installed.